libs/designsystem/src/lib/components/card/card-header/card-header.component.scss

```
@@ -23,3 +24,33 @@ h4 {
   margin: 0;
   font-weight: font-weight('normal');
 }
+
+// Flagged card headers
+
+$notification-levels: ('info');
+
+@each $color-name, $color-value in $notification-colors {
+  $notification-levels: append($notification-levels, $color-name);
+}
+
+@each $level in $notification-levels {
+  $color-name: $level;
+
+  @if $level == 'info' {
+    $color-name: 'semi-light';
+  }
+
+  :host(.#{$level}) {
+    --kirby-card-header-background-color: #{get-color($color-name)};
+    --kirby-card-header-color: #{get-color($color-name + '-contrast')};
```

Consider simplifying:
```suggestion
  $notification-levels: ('info': 'semi-light');
  @each $color-name, $color-value in $notification-colors
{
 $notification-levels: map-merge(
 ( $color-name: $color-name ),
 $notification-levels
 );
 }
 @each $level, $color-name in $notification-levels
 (
  :host(.#{$level}) {
    --kirby-card-header-background-color: #{get-color($color-name)};
    --kirby-card-header-color: #{get-color($color-name + '-contrast')};